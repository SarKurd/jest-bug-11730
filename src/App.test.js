import React from "react";
import { render } from "@testing-library/react";

import App from "./App";

test("should generate snapshots", () => {
  const { container } = render(<App />);

  expect(container).toMatchSnapshot();
  // uncomment this
  // expect(container).toMatchInlineSnapshot();
});
