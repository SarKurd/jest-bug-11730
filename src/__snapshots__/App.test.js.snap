// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should generate snapshots 1`] = `
<div>
  <div>
    Hello world
  </div>
</div>
`;
